### ISSUE TITLE
Automatic color is not marked up as being selected

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#4725]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/ckeditor-4/demo/#document
2. Open console.
3. [Emulate a focused page.](https://www.algolia.com/doc/ui-libraries/autocomplete/guides/debugging/#emulating-a-focused-page-in-chrome-devtools)
4. Click "Text Color" button.
5. Inspect "Automatic" color button.

### Expected result

The button should have `[aria-selected=true]` attribute.

### Actual result

The button does not have `[aria-selected]` attribute.

## Other details

* Browser: all
* OS: all
* CKEditor version: 4.10+
* Installed CKEditor plugins: colorbutton
